src/site/_filters/strip.js

```diff
@@ -18,15 +18,15 @@
  * Removes any characters that should not exist in a string.
```

Comment:
can you update this comment say "for presentation", rather than for any other reason?